docs/docs/cloud/how-tos/stateless_runs.md

```diff
@@ -0,0 +1,180 @@
+# Stateless Runs
+
+Most of the time, you provide a `thread_id` to your client when you run your graph in order to keep track of prior runs through the persistent state implemented in LangGraph Cloud. However, if you have your own database to save runs and don't need to use the built in persistent state, you can create stateless runs.
```

Comment:
`However, if you have your own database to save runs` i don't think that's really the reason people use it -- let's change to just "if you don't need to persist the runs" or the like